email.js

```diff
@@ -16,3 +16,33 @@ function valedateemail() {
     return false;
   }
 }
+
+const nameInput = document.getElementById('name');
+const emailInput = document.getElementById('email');
+const textareaInput = document.getElementById('message');
+
+var forminfo = JSON.parse(localStorage.getItem("forminfo"));
+
+
+if (forminfo){
+nameInput.value = forminfo.name;
+emailInput.value = forminfo.email;
+textareaInput.value = forminfo.textarea;
+}
+
+function storetext() {
+  const name = nameInput.value;
+  const email = emailInput.value;
+  const textarea = textareaInput.value;
+
+  if (!name || !email || !textarea) {
+    return;
+  }
```

Comment:
- [ ] Great implementation of the storage functionality. It is working almost perfectly, but if the user writes only one or two of the inputs, the storage doesn't get updated due to this conditional. I think that the `storetext` function would be better without it, but changing the conditional `||` to `&&` would make it work if the user makes changes to any input. Kindly change that.
 -
<img width="732" alt="image" src="https://user-images.githubusercontent.com/61919617/195647095-ab84607c-afc3-4395-9433-1db7d6de7787.png">

Reply:
thank you for the suggestion